README.md

```diff
@@ -216,6 +216,15 @@ Install packages:
     To activate this project's virtualenv, run the following:
     $ pipenv shell
 
+Install from git:
+
+    $ pipenv install -e git+https://github.com/requests/requests.git@v2.19.1#egg=requests
+    Installing -e git+https://github.com/requests/requests.git@v2.19.1#egg=requests…
+    ...
+    Adding -e git+https://github.com/requests/requests.git@v2.19.1#egg=requests to Pipfile's [packages]...
+
+You can also change the tag/ref (or exclude to get the latest version) by adding @version-or-tag before #egg=project_name
```

Comment:
Would you mind doing a touch of cleanup here and adding a section that describes the syntax including the sections of the url? Suggestion below:

Can you add some syntax highlighting here at the bottom? Then you can end the line with a period.

````markdown
Installing from git:

You can install packages with pipenv from git and other version control systems using URLs formatted according to the following rule:

```
    <vcs_type>+<scheme>://<location>/<user_or_organizatoin>/<repository>@<branch_or_tag>#<package_name>
```

The only optional section is the `@<branch_or_tag>` section.  When using git over SSH, you may use the shorthand vcs and scheme alias `git+git@<location>:<user_or_organization>/<repository>@<branch_or_tag>#<package_name>`. Note that this is translated to `git+ssh://git@<location>` when parsed.

Valid values for `<vcs_type>` include `git`, `bzr`, `svn`, and `hg`.  Valid values for `<scheme>` include `http,`, `https`, `ssh`, and `file`.  In specific cases you also have access to other schemes: `svn` may be combined with `svn` as a scheme, and `bzr` can be combined with `sftp` and `lp`.

Note that it is **strongly recommended** that you install any version-controlled dependencies in editable mode in order to ensure that dependency resolution can be performed with an up to date copy of the repository each time it is performed, and that it includes all known dependencies.

Below is an example usage which installs the git repository located at `https://github.com/requests/requests.git` from tag `v2.19.1` as package name `requests`:

```console
    $ pipenv install -e git+https://github.com/requests/requests.git@v2.19#egg=requests
    Creating a Pipfile for this project...
    Installing -e git+https://github.com/requests/requests.git@v2.19.1#egg=requests...
    [...snipped...]
    Adding -e git+https://github.com/requests/requests.git@v2.19.1#egg=requests to Pipfile's [packages]...
    [...]
```
````